### ISSUE TITLE
New-UDTextbox -Variant outlined with -Mask does not show outlined

### ISSUE THREAD

**Issue by Omzig** [ironmansoftware/powershell-universal#2839]
### Version

4.1.8

### Severity

Low

### Steps to Reproduce

The Texbox variant of outlined does not work when using -Mask, mask apparently disables the variant or defaults it to something else. 
```powershell
New-UDTextbox -Label 'Lead ABA:' -Value $msgInfoData.Aba @disabled -Variant outlined -Mask '000000000' -OnChange {
    # $page:ABA = ([int]$eventdata).ToString("000000000")
    $page:ABA = $Body.Trim('"') # you cannot use eventdata until sometime after v4.1.8
}
```

### Expected behavior

```console
Outlined Variant
```

### Actual behavior

```console
Not Outlined Variant
```

### Visuals

![image](https://github.com/ironmansoftware/issues/assets/1987296/e071d85b-b5f9-4650-97ee-df5ef9cd9758)

**Comment by donlent** [comment]
Duplicate of my issue here:
https://github.com/ironmansoftware/issues/issues/2807

This has already been adressed and resolved for 4.2 😉

**Comment by mattselle** [comment]
Just double checked and this has been fixed
